.github/workflows/api.yml

```diff
@@ -21,33 +21,8 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '16'
-      - uses: bahmutov/npm-install@v1
-      - run: npx playwright install-deps
-      - run: yarn test:api
-        env:
-          DATABASE_URL: http://localhost:3000
-          DATABASE_TOKEN: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJzdXBhYmFzZSIsImlhdCI6MTYwMzk2ODgzNCwiZXhwIjoyNTUwNjUzNjM0LCJyb2xlIjoic2VydmljZV9yb2xlIn0.necIJaiP7X2T2QjGeV-FhpkizcNTX8HjDDBAxpgQTEI
-          DATABASE_CONNECTION: postgresql://postgres:postgres@localhost:5432/postgres
-  deploy-dev:
-    name: Deploy Dev
-    if: github.event_name == 'pull_request' && github.ref != 'refs/heads/main'
-    runs-on: ubuntu-latest
-    needs: test
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: '16'
-      - uses: bahmutov/npm-install@v1
-      - name: Publish app
-        if: env.CF_API_TOKEN
-        uses: cloudflare/wrangler-action@1.3.0
-        env:
-          SENTRY_TOKEN: ${{secrets.SENTRY_TOKEN}}
-          CF_API_TOKEN: ${{secrets.CF_API_TOKEN}}
-        with:
-          apiToken: ${{secrets.CF_API_TOKEN }}
-          workingDirectory: 'packages/api'
+      - run: 'cp .env.tpl .env'
+      - run: yarn test:api:docker
```

Comment:
It's possible that @redaphid and I were confused about deploy-dev. We both thought it was a remnant of the local-tunnel days, where you had to deploy to cloudflare in order to run the dev environment at all.

If it's still useful as an ad-hoc "not quite staging" env we can bring it back.

Reply:
> If it's still useful as an ad-hoc "not quite staging" env we can bring it back.

It is, but it will be re-written with new commits in other PRs which is annoying. Would love to be able to trigger it in CI for a specific branch, but I think we won't be able to provide a branch on a manual workflow trigger

Reply:
yes please keep the deploy-dev job and also this yaml is not valid at all i think.